integration-tests/tests/network/ban_peers.rs

```diff
@@ -1,8 +1,6 @@
-pub use runner::*;
+pub use crate::runner::*;
 use std::time::Duration;
 
-mod runner;
```

Comment:
:+1: we no longer need to recompile the `runner` module for each test over and over again!